modules/ballerina-native/src/main/java/org/ballerinalang/nativeimpl/connectors/data/sql/Constants.java

```diff
@@ -58,8 +58,6 @@
      * Constants for SQL DataTypes.
      */
     public static final class SQLDataTypes {
-        public static final String LONG = "LONG";
```

Comment:
In the current implementation, only the standard jdbc sql types[1] can be used to define the type of the input/output parameters. Long, String etc are not generic sql types and those should be defined as BIGINT, VARCHAR. But still for any "Long" type supported DB, we can use BIGINT as parameter type when defining the parameter and get the same result.  In future we can add those DB specific types if needed. 
[1] http://docs.oracle.com/javase/6/docs/api/java/sql/Types.html